src/main/java/team/chisel/client/render/SubmapManagerGlotek.java

```diff
@@ -0,0 +1,80 @@
+package team.chisel.client.render;
+
+import net.minecraft.block.Block;
+import net.minecraft.client.renderer.RenderBlocks;
+import net.minecraft.client.renderer.texture.IIconRegister;
+import net.minecraft.util.IIcon;
+import net.minecraft.world.IBlockAccess;
+
+import team.chisel.ctmlib.RenderBlocksCTM;
+import team.chisel.ctmlib.TextureSubmap;
+import cpw.mods.fml.relauncher.Side;
+import cpw.mods.fml.relauncher.SideOnly;
+
+public class SubmapManagerGlotek extends SubmapManagerBase {
+
+    @SideOnly(Side.CLIENT)
```

Comment:
Don't use `@SideOnly

Reply:
so this I just copied from the already existing anti block class, dont know what I should use instead

Reply:
Nothing, it's not needed